src/test/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+server.port=2001
```

Comment:
we now need explicit test properties, as the test db url can no longer be hard coded in the common code